### DISCUSSION TITLE
【doris 1.2.7】为什么我的内存一直不释放？

### DISCUSSION THREAD

**Discussion by zy18509510870** [apache/doris#29247] [A - General / Q&A]
![Snipaste_2023-12-28_18-09-37](https://github.com/apache/doris/assets/142372426/844cbab2-b4de-403a-845b-5c859bf12496)
现在是两台机器,fe跟be部署到一台,另外一台部署fe(Observer),be,两个机器内存都是16G，之前内存限制一直是1G，然后前几天报错,说是内存超限，然后我调到2G，过了一天有报错一直到现在调到8G

**Comment by zy18509510870** [comment]
![1](https://github.com/apache/doris/assets/142372426/522fc133-ef9f-4e80-b45f-7abcd63bb13c)
这s是我内存超限的报错

**Reply by LemonLiTree** [reply]
系统可用内存太少了
